pynamodb/models.py

```diff
@@ -169,40 +171,104 @@ class MetaModel(AttributeContainerMeta):
     """
     def __init__(cls, name, bases, attrs):
         super(MetaModel, cls).__init__(name, bases, attrs)
+
+        # We don't' allow inheritance from multiple direct parents
+        if (len(bases) > 1):
+            raise InheritanceError("Model can't inherit from multiple direct parents")
+
+        parent = None if len(bases) == 0 else bases[0]
+        # TODO: special parents as constants
+        parent_is_attribute_container = parent.__name__ == 'AttributeContainer'
+        parent_is_model = parent.__name__ == 'Model'
+        parent_meta_class = None
+        is_parent_abstract = None
+        if parent  and not parent_is_attribute_container:
+            parent_meta_class = getattr(parent, META_CLASS_NAME, None)
+        if parent_meta_class and hasattr(parent_meta_class, ABSTRACT):
+            is_parent_abstract = getattr(parent_meta_class, ABSTRACT)
+
+        # Temporary check until implementing other inheritance types
+        if parent and not parent_is_attribute_container and not parent_is_model and not is_parent_abstract:
+            raise NotImplementedError("Inheritance from non abstract models is not supported now")
+
         if isinstance(attrs, dict):
+            meta_class = attrs.get(META_CLASS_NAME, None)
```

Comment:
open question now that we're starting to allow inheritance:
Do we allow models to inherit the `Meta` class from their parent class(es) or do we require each Model define their own `Meta` class? If the latter, how do we plan on handling `DefaultMeta` since we currently allow Models that do not define `Meta` at all?